lint.whitelist

```diff
@@ -827,3 +827,14 @@ MISSING DEPENDENCY: web-nfc/resources/nfc-helpers.js
 MISSING DEPENDENCY: shape-detection/resources/shapedetection-helpers.js
 MISSING DEPENDENCY: webxr/resources/webxr_util.js
 MISSING DEPENDENCY: contacts/resources/helpers.js
+
+# Tests that are false positives for using Ahem as a system font
+AHEM SYSTEM FONT: acid/acid3/test.html
+AHEM SYSTEM FONT: preload/single-download-preload.html
```

Comment:
This test doesn't even use Ahem, perhaps s/ahem/Foo/ or something to avoid the lint exception?

Reply:
Good point, done.